pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 125
+include = '\.pyi?$'
```

Comment:
This is added to allow editors that support `black` for auto-formatting to align with the `pre-commit` configuration for `black`.

Reply:
I see many lines of what looks like only formatting changes, and some of those don't even hit this line length of 125. Are those all intentional, or just happened before adding this?

Reply:
@tvainika These are all intentional from running `make lint`.

Reply:
Just running `make lint` on current main branch does not trigger those reformattings (like with `parser.add_argument`), but those are caused by adding trailing commas. Those cause much more than necessary formatting changes here, not only running `make lint`. Not objecting having those all now, but generally I feel that formatting changes are better to separate.